src/main/java/org/littleshoot/proxy/BadGatewayFailureHttpResponseComposer.java

```diff
@@ -0,0 +1,37 @@
+package org.littleshoot.proxy;
```

Comment:
The code in this class is copied from `writeBadRequest` method of `ClientToProxyConnection` class